### PR TITLE
Session timeout RH link defect

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Fixes session timeout page link to take the base url from the cookie if it exists.

*This PR has been branched off my other PR (#1070) because some changes needed to be made to that PR too for this*

### How to review
Open a schema with a `Account Service URL` set. Check that the session expired page uses this as a base url for the page link.

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by rmccar** [comment]
[Defect: Link to re-enter AUC following a session timeout in pre-prod and Int are pointing to Prod RH-UI](https://trello.com/c/Mtpxe4HM/5638-defect-link-to-re-enter-auc-following-a-session-timeout-in-pre-prod-and-int-are-pointing-to-prod-rh-ui)

**Comment by kylelawsonAND** [comment]
Link appears to redirect correctly locally (no page to load of course but url is correct I think). Can't test in a different env but otherwise all looks sensible, unofficial ✅  from me

**Review by petechd — CHANGES_REQUESTED** [review]
You need to change the base branch for this PR to main.

**Comment by rmccar** [comment]
> You need to change the base branch for this PR to main.

Yeah I know I will change it when my other PR goes in. Because I branched off my other PR this allows me to see the changes without the ones from that branch at the moment

File: app/routes/errors.py [hunk 1]
```diff
@@ -52,7 +53,9 @@ def _render_error_page(status_code, template=None, **kwargs):
     handle_language()
     business_survey_config = get_survey_config(theme=SurveyType.BUSINESS)
     other_survey_config = get_survey_config(
-        theme=SurveyType.SOCIAL, base_url=ACCOUNT_SERVICE_BASE_URL_SOCIAL
+        theme=SurveyType.SOCIAL,
+        base_url=cookie_session.get("account_service_base_url")
+        or ACCOUNT_SERVICE_BASE_URL_SOCIAL,
     )
```
Comment (MebinAbraham) [inline app/routes/errors.py:59]: Actually this won't work as the base url will be used by both types in this case. We actually don't need any changes in runner. You need to update the runner terraform and eq pipelines to add in `ACCOUNT_SERVICE_BASE_URL_SOCIAL`, at the moment, it doesn't set it, therefore it defaults to prod.

**Pull request closed without merging**